swagger_parser/README.md

```diff
@@ -15,33 +15,70 @@
 - Support for multiple languages (Dart, Kotlin)
 - Web interface at https://carapacik.github.io/swagger_parser
 
-### Run command
+## Installation
 
-To generate boilerplate code, run the `generate` program inside directory where your `pubspec.yaml` file is located:
-```shell
-dart run swagger_parser:generate
-```
+### Add dependency
 
-Add your OpenApi json file configuration to your `pubspec.yaml` or create a new config file called `swagger_parser.yaml`.
-An example of YAML is shown below
+In your pubspec.yaml, add the following dependencies:
 
-### Configure package
 ```yaml
+dependencies:
+  # ...
+  dio: ^4.0.0
+  json_annotation: ^4.7.0
+  retrofit: ^3.3.0
+
 dev_dependencies:
+  # ...
   swagger_parser: ^0.3.0
+  build_runner: ^2.3.0
+  json_serializable: ^6.5.0
+  retrofit_generator: ^4.2.0
+```
+
+### Create swagger_parser.yaml
 
+This is the configuration file that will be read for running the parser according to your needs.
+
+```yaml
 swagger_parser:
-  json_path: assets/swagger.json # Required. Sets the json path directory for generated files
-  output_directory: generated # Required. Sets output directory for generated files
+  json_path: api_definition/openapi.json # Required. Sets the json path directory for api definition. This is just a suggestion for path
+  output_directory: lib/api # Required. Sets output directory for generated files (api clients and models)
   language: dart # Optional. Sets the programming language. Current available languages are: dart, kotlin. Default: dart
   squish_clients: true # Optional. Set 'true' to put all clients in one folder. Default: false
-  client_postfix: Client # Optional. Set postfix for client folder and Class. Works if there is only a single class or squish is true. Default: Client
+  client_postfix: ApiClient # Optional. Set postfix for client folder and Class. Works if there is only a single class or squish is true. Default: Client
   freezed: false # Optional (dart only). Set 'true' to generate data classes using freezed package. Default: false
 ```
 
+### Compile your generator locally
+
+If you are using Flutter, you can replace all "dart run" calls with "flutter pub run".
+
+To generate boilerplate code, run the `generate` program inside directory where your `pubspec.yaml` file is located:
+```shell
+dart run swagger_parser:generate
+```
+
+Add your OpenApi json file configuration to your `pubspec.yaml` or create a new config file called `swagger_parser.yaml`.
+An example of YAML is shown below
+
 If you name your configuration file something other than `swagger_parser.yaml` or `pubspec.yaml` 
 you will need to specify the name of the YAML file as an argument.
 
 ```shell
 dart run swagger_parser:generate -f <path to your config file>
 ```
+
```

Comment:
From this line and below, I think it is not necessary to add, because any developer will understand that next comes the generation of code from retrofit, json_serializable and freezed.

Reply:
That's not entirely correct, lot's of people may not know those tools and not be aware of this command being needed. I would leave it as it's not causing any harm and may help someone.